apps/prairielearn/src/pages/instructorAssessments/instructorAssessments.html.tsx

```diff
@@ -196,3 +199,189 @@ export function AssessmentStats({ row }: { row: AssessmentStatsRow }) {
     </td>
   `;
 }
+
+export function PreactInstructorAssessments({
+  resLocals,
+  rows,
+  assessmentIdsNeedingStatsUpdate,
+  csvFilename,
+}: {
+  resLocals: Record<string, any>;
+  rows: AssessmentRow[];
+  assessmentIdsNeedingStatsUpdate: string[];
+  csvFilename: string;
+}) {
+  const { urlPrefix, authz_data, course, __csrf_token } = resLocals;
+
+  return renderHtmlDocument(
+    <html lang="en">
+      <head>
+        <PreactHeadContents resLocals={resLocals} />
+        <script src={compiledScriptPath('instructorAssessmentsClient.ts')}></script>
```

Comment:
TODO: we'd want to mount a `AssessmentsTable` component on the client to handle the loading here.